docusaurus.config.ts

```diff
@@ -59,6 +59,7 @@ const config: Config = {
           // Please change this to your repo.
           // Remove this to remove the "edit this page" links.
           editUrl: 'https://github.com/ajnart/homarr-docs/edit/master',
+          authorsMapPath: "authors.yml"
```

Comment:
Is this path correct? It's placed in `blog` directory

Reply:
Yes, it seems to be correct. The build passes without warnings but with blog/authors.yml it will emit warnings